<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btnListen">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>监听</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtReceiveInformation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>300</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblConnection">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No Connection</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>421</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>历史记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_selectSql">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>显示历史记录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>实时数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Abnshow">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>显示异常记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delAll">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清除所有数据</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
